error/s2n_errno.h

```diff
@@ -364,14 +387,15 @@ extern __thread const char *s2n_debug_str;
 
 /** Calculate and print stacktraces */
 struct s2n_stacktrace {
-    char **trace;
+    char** trace;
```

Comment:
nit - I think clang-format usually moves the pointers the other way?

Reply:
This is what running clang format did. The lint job passed as well, so it doesn't seem to care? No idea.